### ISSUE TITLE
Suggest to add AllContributors to better show contributions from the community

### ISSUE THREAD

**Issue by eduardo-rodrigues** [HSF/prmon#246]
See an example at the bottom of the README https://github.com/scikit-hep/uproot5 or at many other Scikit-HEP repos.

More info at https://allcontributors.org/.

**Comment by Pratz2005** [comment]
Hello, can I work on this issue

**Comment by graeme-a-stewart** [comment]
Thanks @eduardo-rodrigues - it's a good suggestion and it's not too hard to do.

**Comment by graeme-a-stewart** [comment]
@all-contributors please add @amete for infrastructure, tests and code
@all-contributors please add @quantum-shift for tests and code
@all-contributors please add @chuckatkins for infrastructure
@all-contributors please add @Cossack42 for code

**Comment by graeme-a-stewart** [comment]
@all-contributors
please add @quantum-shift for tests and code
please add @chuckatkins for infrastructure
please add @Cossack42 for code
please add @chrisburr for code
please add @rmaganza for code
please add @miguelgila for code
please add @protopopescu for code

**Comment by graeme-a-stewart** [comment]
I can't seem to get All Contributors to add more than one person at a time 😕

**Comment by graeme-a-stewart** [comment]
@all-contributors please add @chuckatkins for infrastructure. please add @Cossack42 for code. please add @chrisburr for code. please add @rmaganza for code. please add @miguelgila for code. please add @protopopescu for code.

**Comment by graeme-a-stewart** [comment]
https://github.com/all-contributors please add @Cossack42 for code.

**Comment by graeme-a-stewart** [comment]
@all-contributors please add @Cossack42 for code.

**Comment by eduardo-rodrigues** [comment]
Hi @graeme-a-stewart, I seem to recall that I had the same issue. Basically I was writing the magic instruction one at the time per person in an issue or PR, and get that merged asap (no need to wait for the CI). It took a few minutes, yes.

**Comment by graeme-a-stewart** [comment]
> Hi [@graeme-a-stewart](https://github.com/graeme-a-stewart), I seem to recall that I had the same issue. Basically I was writing the magic instruction one at the time per person in an issue or PR, and get that merged asap (no need to wait for the CI). It took a few minutes, yes.

Yeah, it's a bit tedious, but after we are caught up it will be a lot easier to maintain!

**Comment by graeme-a-stewart** [comment]
@all-contributors please add @miguelgila  for code.

**Comment by graeme-a-stewart** [comment]
@all-contributors please add @protopopescu  for code.

**Comment by graeme-a-stewart** [comment]
Done! Thanks again for the hint, @eduardo-rodrigues.
